Voting and ToR discussion

woodbe opened the pull request:
I think we should update the ToR voting section to make it clear that the voting description there is for iTC-related votes. What I am intending here is that this would be for things impacting the iTC as a whole, and not specifically for things like voting on the documents. That isn't to say that this couldn't be used for those things, just that the expectation would be that the iTC may have other voting parameters for those instead.

So what I'm thinking is along the lines added here to make this more clear. We would then work on something else to define the classes of voting in a separate document outside the ToR.

Commit:
Voting and ToR discussion

I think we should update the ToR voting section to make it clear that the voting description there is for iTC-related votes. What I am intending here is that this would be for things impacting the iTC as a whole, and not specifically for things like voting on the documents. That isn't to say that this couldn't be used for those things, just that the expectation would be that the iTC may have other voting parameters for those instead.

So what I'm thinking is along the lines added here to make this more clear. We would then work on something else to define the classes of voting in a separate document outside the ToR.

ansukert reviewed (approved):
I agree with the change. Just two things:
1. May want to define in some way what you mean by an "itc issue" so it is put in the proper context.
2. I am not really sure we need the bureaucracy of a voting process for documents; in all the 15 years I have been involved in developing PPs for HCDs we have never had to vote on document content. It is usually a case where either there is consensus (95+% of the time) or in the rare case there has been a disagreement the chair made the call. Let's not create a big process where one is not needed. I think what I just described is all we need - strive for consensus and if not the iTC chair is the tie-breaker.

woodbe commented:
@ansukert So normally I would agree about the voting, but the ND iTC has created several levels of voting in their group, and that may be good or it may be overkill (probably overkill).

I think in most cases, the voting would be noted in the Issue (i.e. that it was approved on some call/meting/email) and then a PR would be voted upon by the approval process in GitHub. The ND iTC didn't have GitHub for this when they started, and I think they have gone in a way that makes things very heavy (I had heard they have a LOT of votes, though again, having GitHub would probably take a lot of that away).

As for the "iTC issues" I tried to lay that out with examples (changes to the ESR, publication of documents), but maybe we can come up with a better list or way to state it.

In the end my point was really to provide an explicit way to state that this is the "default" voting method, but it doesn't need to be the only one, and then we could provide some examples, with recommendations surrounding GitHub (i.e. what would we recommend for people using GitHub).

To some extent that would be an expansion of the recommendation already made about setting up the Working and Master approval process, and maybe that is enough.

Commit:
further updates to the ToR

To address @ansukert comments, I have made some additional clarifications about the voting. I have also modified some of the admonitions to use GUIDANCE and REVIEW for things that don't need to be in the published document

woodbe commented:
@ansukert I have tried to make some further updates to address your comments about better examples. Please take a look at the latest update.
